Sources/Testing/Issues/Issue.swift

```diff
@@ -101,8 +100,12 @@ public struct Issue: Sendable {
   }
 
   /// The severity of this issue.
-  @_spi(Experimental)
   public var severity: Severity
+  
+  /// If the issues is a failing issue.
+  public var isFailure: Bool {
+    severity == .error
+  }
```

Comment:
I suggest adding this `isFailure` property via a separate PR as `@_spi` first, so that this PR can consist only of _removals_ of `@_spi` guards and similar affordances.